compatibility/bazel_tools/test_utils/DA/Test/Tar.hs

```diff
@@ -0,0 +1,79 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
I'm wondering how we can avoid duplicating much of `libs-haskell` in the long run. Maybe we should extract those into their own Bazel workspace and import them to both `@daml` and `@compatibility`?

Reply:
Yeah I’m not really sure about that either sadly. For now it still seems to be somewhat manageable and hopefully we don’t need that much more Haskell code for tests so I think it might not be a huge problem but it could become one.